cdp/src/destination-types/handlers.ts

```diff
@@ -0,0 +1,42 @@
+// This file is responsible for handling the destination types API. It's a
+// simple API that returns a list of all the available destination types.
+//
+// The destination types are defined in code for now, but it's possible that we
+// will want to move them to the database in the future to allow dynamic
+// addition of new destination types.
+
+import Koa from 'koa'
+
+type DestinationType = {
+    type: string
+    name: string
+    description: string
+    configSchema: Record<string, unknown> // A JSONSchema describing the configuration
+}
+
+const destinationTypes: { [type: string]: DestinationType } = {
+    webhook: {
+        type: 'webhook',
+        name: 'Webhook',
+        description: 'Send events to a webhook',
+        configSchema: {
```

Comment:
further down the line we'll want to add the ability to set headers too

Reply:
Yep I haven't filled out much on this part yet. I'm going to go deep first and get the stack working end to end e.g.:

 1. get the basic API deployed
 2. get the frontend able to call it

Then we can do follow-ups to add the requirements here.